### ISSUE TITLE
1Password button is not discoverable with VoiceOver

### ISSUE THREAD

**Issue by astralbodies** [wordpress-mobile/WordPress-iOS#3490]
When scrolling left & right with one finger with VoiceOver, the 1Password button is not reachable and not usable.

**Comment by rachelmcr** [comment]
I tested in the latest stable release (5.6.1) and the 1Password button is reachable by swiping right when Username field is selected or swiping left when Password field is selected, as works as expected. Closing this as works for me now.
